cicecore/cicedyn/general/ice_flux.F90

```diff
@@ -405,8 +421,12 @@ subroutine alloc_flux
 
       use ice_grid, only : grid_ice
 
+      logical (kind=log_kind) :: tr_pond
+
       integer (int_kind) :: ierr
 
+      call icepack_query_tracer_flags(tr_pond_out=tr_pond)
```

Comment:
Was the intention to use tr_pond to make the pond diagnostics allocation optional as in init_history_therm?  It doesn't seem to be used here.